catalog/spatial/geocoding/spatial-geocoding-offline-catalog/src/main/java/org/codice/ddf/spatial/geocoding/index/GeoNamesCatalogIndexer.java

```diff
@@ -360,4 +369,68 @@ private void fileProcessingComplete(String resource) {
       LOGGER.debug("Unable to create {} to indicate {} processed", processedIndicator, resource, e);
     }
   }
+
+  private Integer getGeoNameGazetterSortByFeatureClass(GeoEntry geoEntry) {
```

Comment:
✏️ Instead of this large switch, we can invert the relationship to make it OO.  We could create a new interface for getting the sort value.  Another option would be to see how this might fit into the `GeoEntry`.